src/fidesctl/connectors/bigquery.py

```diff
@@ -0,0 +1,33 @@
+from sqlalchemy import create_engine, inspect
+from sqlalchemy.engine import Engine
+
+from fidesctl.connectors.models import BigQueryConfig, ConnectorFailureException
+
+
+def get_bigquery_engine(bigquery_config: BigQueryConfig) -> Engine:
+    """
+    Creates the SQLAlchemy engine for connecting to a BigQuery dataset
+    """
+    project_id = bigquery_config.keyfile_creds.project_id
+    dataset = bigquery_config.dataset
+    engine = create_engine(
+        f"bigquery://{project_id}/{dataset}",
+        credentials_info=bigquery_config.keyfile_creds.dict(),
+    )
+    return engine
+
+
+def validate_bigquery_engine(bigquery_config: BigQueryConfig) -> None:
```

Comment:
i think that the engine should be validated at the time of creation. Having a separate function is good design, but I'd also think this should get immediately called from `get_bigquery_engine`